src/main/java/io/quarkus/bot/command/RerunWorkflowCommand.java

```diff
@@ -0,0 +1,56 @@
+package io.quarkus.bot.command;
+
+import io.quarkus.bot.workflow.WorkflowConstants;
+import org.jboss.logging.Logger;
+import org.kohsuke.github.GHIssue;
+import org.kohsuke.github.GHPullRequest;
+import org.kohsuke.github.GHRepository;
+import org.kohsuke.github.GHWorkflowRun;
+
+import javax.enterprise.context.ApplicationScoped;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Optional;
+
+@ApplicationScoped
+public class RerunWorkflowCommand implements Command {
+
+    private static final Logger LOG = Logger.getLogger(RerunWorkflowCommand.class);
+
+    @Override
+    public List<String> labels() {
+        return Arrays.asList("test", "retest");
+    }
+
+    @Override
+    public void run(GHIssue issue) {
+        try {
+            GHRepository repository = issue.getRepository();
+            GHPullRequest pullRequest = repository.getPullRequest(issue.getNumber());
+
+            List<GHWorkflowRun> ghWorkflowRuns = repository
+                    .queryWorkflowRuns()
+                    .branch(pullRequest.getHead().getRef())
+                    .status(GHWorkflowRun.Status.COMPLETED)
+                    .list().toList();
+
+            Optional<GHWorkflowRun> ghWorkflowRun = ghWorkflowRuns.stream()
+                    .filter(workflowRun -> WorkflowConstants.QUARKUS_CI_WORKFLOW_NAME.equals(workflowRun.getName()))
+                    .filter(workflowRun -> workflowRun.getHeadRepository().getOwnerName()
+                            .equals(pullRequest.getHead().getRepository().getOwnerName()))
+                    .sorted(Comparator.comparingLong(GHWorkflowRun::getId))
+                    .reduce((run, last) -> last);
+
+            if (ghWorkflowRun.isPresent()) {
+                LOG.info("Rerunning workflow: " + ghWorkflowRun.get().getHtmlUrl());
+                ghWorkflowRun.get().rerun();
+            } else {
+                LOG.warn("Unable to find a workflow run to rerun");
```

Comment:
I think you should use the reaction feature here, similarly to what Dependabot does with the commands (i.e. add a reaction to the comment). See `GHIssueComment#createReaction()`.

You would either have to push the issue comment here or you could return a boolean (or a `ReactionContent`) from the run method and centralize it in the other class. I think the latter would probably be better.